src/client/src/pages/Tasks/Tasks.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+import Header from "../../components/Header/Header.jsx";
+import AddTaskForm from "../../components/AddTaskForm/AddTaskForm.jsx";
+import TasksList from "../../components/TasksList/TasksList.jsx";
+import EmptyListNote from "../../components/EmptyListNote/EmptyListNote.jsx";
+import ActionBar from "../../components/ActionBar/ActionBar.jsx";
+import RemoveAllBtn from "../../components/RemoveAllButton/RemoveAllBtn.jsx";
+import { Toast } from "monday-ui-react-core";
+import "./tasks.css";
+
+const Tasks = ({ tasks, setTasks, taskService }) => {
+  const [editTask, setEditTask] = useState(null);
+  const [searchInput, setSearchInput] = useState("");
+  const [statusFilter, setStatusFilter] = useState("all");
+  const [presentedTasksNum, setPresentedTasksNum] = useState(0);
```

Comment:
I guess that the default here should be tasks.length and not 0, you're just "counting" on some other component to change that, in this case the "TasksList" component